requests/models.py

```diff
@@ -577,6 +577,10 @@ def prepare_hooks(self, hooks):
         for event in hooks:
             self.register_hook(event, hooks[event])
 
+    def send(self, session):
```

Comment:
I see no difference, except that one is more intuitive for people already using that API. Sessions send requests, requests don't send sessions.